### DISCUSSION TITLE
Got "SyntaxError: Cannot use import statement outside a module" when loading an internal package from nest.js

### DISCUSSION THREAD

**Discussion by njiang** [vercel/turborepo#6362] [Help] [answered]
Hi, 

I'm developing a web app with next.js frontend and several nest.js backends with turborepo.  I created an internal package to share the logging logic between backends.  The package imports from nest-winston.  I referenced the package within my nest.js backend.  When i do nest start i got this error: SyntaxError: Cannot use import statement outside a module.  I think it's due to the package is not transpiled.  Not sure what configurations I need to set to address it? 

Thanks!

**Comment by anthonyshew** [comment]
You're correct that your dependency code is not being transpiled. Internal Packages assume that you are using a bundler for your final application so you may want to try that!

**Reply by anthonyshew** [reply, marked as answer]
We released some documentation today that should be able to help further: https://turbo.build/repo/docs/crafting-your-repository/creating-an-internal-package
